src/area-select.lite.tsx

```diff
@@ -21,7 +21,6 @@ export const AreaSelect = ({
       dropdownMatchSelectWidth={false}
       optionLabelProp="label"
       filterOption={filterOption}
-      filterSort={filterSort}
```

Comment:
If I search an area, what sorting will it be? We'd better to keep the consistency here.

Reply:
What do you mean? We decided to get rid of it in previous comment.

Reply:
I haven't checked the lite implementation, but the areas array, that it receives from the Context is already sorted. So there is no need of additional job. In other words, sorting approach is now exact the same for both versions of widget.

Plus sorting can be always altered via areaSorter config prop, as described in [documentation](https://boyuai.github.io/antd-country-phone-input/demos/custom-sorter).

Reply:
I see. Could you provide some screenshots about that(no need to check the lite version)? Let's see if the sorting is still good here.

Reply:
Cool!